FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -315,7 +315,7 @@ NS_SWIFT_NAME(Auth)
 @property(nonatomic, strong, readonly) FIRAuthNotificationManager *notificationManager;
 
 @property(nonatomic, strong) NSString *firebaseAppName;
-@property(nonatomic) FIRAuthKeychainServices *keychainServices;
+@property(nonatomic) id<FIRAuthStorage> keychainServices;
```

Comment:
This was the real culprit to the failures I was seeing. Was having a very difficult time debugging until I stumbled on this. The original diff prevented the injected type from functioning– it would just default to the real `AuthKeychainServices` type.